tests/unit/utils/test_arxiv.py

```diff
@@ -78,3 +80,28 @@ def test_extract_arxiv_additional_chars():
     """
     with raises(UnicodeEncodeError):
         clean_arxiv(u'"1808.01899\u201c')
+
+@mark.vcr
```

Comment:
`vcr` should add cassettes if I'm not mistaken, did we add the `xml` files manually?

Reply:
You are correct, there are no cassettes ymls... I wasn't able to run tests locally, because I didn't want to spend time setting up the environment. About xmls, I added them but, I am not using them in vcr.

Reply:
I know I understand, since we don't have cassettes, these tests are not functioning as we want, because we don't have response save, and it will be generated just on CI